src/domain/collaboration/callout/CalloutSummary.tsx

```diff
@@ -35,12 +35,6 @@ export const CalloutSummary: FC<{
           {callout?.profile.description}
         </Typography>
       </Box>
-      <Box>
```

Comment:
This is just a change Simone asked me yesterday, just removing a field in a summary window